rust/windows-client/src-tauri/src/client/updates.rs

```diff
@@ -0,0 +1,132 @@
+//! Module to check the Github repo for new releases
+
+use crate::client::about::get_cargo_version;
+use std::str::FromStr;
+use url::Url;
+
+#[derive(Debug, serde::Deserialize)]
+pub(crate) struct Release {
+    /// e.g. <https://github.com/firezone/firezone/releases/tag/1.0.0-pre.8>
```

Comment:
Could we link directly to the MSI installer package? I'm thinking of non-technical folks who may have GitHub phobia

Reply:
E.g. https://github.com/firezone/firezone/releases/download/1.0.0-pre.8/windows-client-x64.msi